### PR TITLE
Fix #1817 422 instead of 404

### LINKED ISSUE #1817: [BUG] in POST v2/enitities/<id> with unknown id, the error code is not the expected
in POST v2/enitities/<id> with unknown id, the error code is not the expected.

**InvalidModification (422)** is returned if some piece of information is missing in payload. The expected value is **NotFound (404)** because the ID referred in the request has not been found.

#### Update Request 
```
 POST http://localhost:1026/v2/entities/house
 Content-Type: application/json
 Fiware-Service: test_update_entity_id
 Fiware-ServicePath: /test
 {"humidity": {"value": "80"}}
```

#### Update Response 
```
  http code: 422
 date: Mon, 22 Feb 2016 09:40:12 GMT
 connection: Keep-Alive
 content-type: application/json
 content-length: 69
 {"error":"InvalidModification","description":"Entity does not exist"}
```
#### Expected Response 
```
  http code: 404
 {"error":"NotFound","description":"No context element found"}
```

### PR DESCRIPTION
**Pull request by crbrox**
Fixes #1817 

Returning 422 InvalidModification instead of 404 NotFound whrn POSTing entity

@kzangeli 
@fgalan

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -26,3 +26,4 @@
 - Add: options=values mode for entity queries in NGSIV2 (Issue #1049)
 - Fix: some of the operations reported an incorrect 'Allow' HTTP Header on Bad Verb, now fixed
 - Add: sorted URI param in NGSIv2 queries to sort entities by attribute (Issue #1103)
+- Fix: Returning 422 InvalidModification instead of 404 NotFound whrn POSTing entity (Issue #1817)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: whrn -> when

**Comment by kzangeli** [comment]
LGTM

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (crbrox) [inline CHANGES_NEXT_RELEASE]: Fixed bd46e4c

**Comment by fgalan** [comment]
LGTM

**Pull request merged**